plugins/backend/catalog-backend-module-aws/src/utils/tags.ts

```diff
@@ -71,8 +71,11 @@ export const labelsFromTags = (tags?: Tag[] | Record<string, string>) => {
     )
     .reduce((acc: Record<string, string>, [key, value]) => {
       if (key && value) {
-        const k = key.replaceAll(':', '_').replaceAll('/', '-');
-        acc[k] = value.replaceAll('/', '-').substring(0, 63);
+        let k = key.replaceAll(':', '_').replaceAll('/', '-');
```

Comment:
I do wonder if [slugify](https://www.npmjs.com/package/slugify) would drop in here.